### DISCUSSION TITLE
Is request methods and Uri etc. highlighted by lspsaga?

### DISCUSSION THREAD

**Discussion by MannyFay** [nvimdev/lspsaga.nvim#1458] [Q&A] [answered]
Hey folks,

does anybody know, if this red/white highlighted text after the name of the function comes from lspsaga plugin in my PHP/Laravel controller file?
![Screenshot 2024-06-02 at 12 23 05](https://github.com/nvimdev/lspsaga.nvim/assets/45234439/63a5a37f-21f7-4d24-811c-3d8d38e1c78f)

**Comment by glepnir** [comment, marked as answer]
Nope

**Reply by MannyFay** [reply]
Thank's! Do you have an idea if it is LSP itself ...or something like that?!?

**Reply by glepnir** [reply]
did you use semantic token ?
